### ISSUE TITLE
Can't compile 9.0.0 version

### ISSUE THREAD

**Issue by Nihisil** [acacode/swagger-typescript-api#247]
Typescript `v4.2.4`.

Command to generate files:
```bash
swagger-typescript-api \
        -p openapi.json \
        -o app/src/client/ \
        --modular \
        --axios \
        --single-http-client
  ```

File `http-client.ts`:
```ts
if (type === ContentType.FormData && body && body !== null && typeof body === "object") {
  requestParams.headers.common = { Accept: "*/*" };
  requestParams.headers.post = {};
  requestParams.headers.put = {};

  const formData = new FormData();
  for (const key in body) {
    formData.append(key, body[key]);
  }
  body = formData;
}
```

Error:
```bash
$ tsc --noEmit && eslint {src,cypress}/**/*.{ts,tsx} --max-warnings=0
src/client/http-client.ts:104:30 - error TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{}'.
  No index signature with a parameter of type 'string' was found on type '{}'.

104         formData.append(key, body[key]);
                                 ~~~~~~~~~
Found 1 error.
```

Are there any workarounds for that?

**Comment by js2me** [comment]
Hello @Nihisil !   
Thank you for bug report, I'll fix that quickly.

**Comment by js2me** [comment]
Fixed in 9.0.1

**Comment by Nihisil** [comment]
Thank you for the fast fix
